katello/hooks/pre_commit/13-hiera.rb

```diff
@@ -29,14 +33,22 @@
   # Check if it's actually 90% of the required. If a crash kernel is enabled
   # then the reported total memory is lower than in reality.
   if facts[:memory][:system][:total_bytes] < (required_memory * 1024 * 1024 * 1024 * 0.9)
-    say "<%= color('Insufficient memory for tuning size', :bad) %>"
-    say "Tuning profile '#{new_tuning}' requires at least #{required_memory} GB of memory and #{required_cores} CPU cores"
+    if module_enabled?('foreman')
```

Comment:
Things look good. I think this could be more a little more robust against the future by checking if `app_value(:tuning)`

Reply:
I will test with this to see how it works.

Reply:
updated now thanks